src/patterns/data/default/response_to_chemical_stimulus_trait.tsv

```diff
@@ -1,143 +1,111 @@
-OBA:2040118		CHEBI:71415	nitrofurantoin	<https://orcid.org/0000-0002-7073-9172>|<https://orcid.org/0000-0001-8314-2140>
-OBA:2040119		CHEBI:71418	triamcinolone acetonide	<https://orcid.org/0000-0002-7073-9172>|<https://orcid.org/0000-0001-8314-2140>
-OBA:2040120		CHEBI:72564	temozolomide	<https://orcid.org/0000-0002-7073-9172>|<https://orcid.org/0000-0001-8314-2140>
-OBA:2040121		CHEBI:75045	dabrafenib	<https://orcid.org/0000-0002-7073-9172>|<https://orcid.org/0000-0001-8314-2140>
-OBA:2040122		CHEBI:75998	trametinib	<https://orcid.org/0000-0002-7073-9172>|<https://orcid.org/0000-0001-8314-2140>
-OBA:2040123		CHEBI:76004	dimethyl fumarate	<https://orcid.org/0000-0002-7073-9172>|<https://orcid.org/0000-0001-8314-2140>
-OBA:2040124		CHEBI:76010	dolutegravir	<https://orcid.org/0000-0002-7073-9172>|<https://orcid.org/0000-0001-8314-2140>
-OBA:2040125		CHEBI:7735	olanzapine	<https://orcid.org/0000-0002-7073-9172>|<https://orcid.org/0000-0001-8314-2140>
-OBA:2040126		CHEBI:7824	oxcarbazepine	<https://orcid.org/0000-0002-7073-9172>|<https://orcid.org/0000-0001-8314-2140>
-OBA:2040127		CHEBI:8028	perphenazine	<https://orcid.org/0000-0002-7073-9172>|<https://orcid.org/0000-0001-8314-2140>
-OBA:2040128		CHEBI:80630	irinotecan	<https://orcid.org/0000-0002-7073-9172>|<https://orcid.org/0000-0001-8314-2140>
-OBA:2040129		CHEBI:8093	phenylephrine	<https://orcid.org/0000-0002-7073-9172>|<https://orcid.org/0000-0001-8314-2140>
-OBA:2040130		CHEBI:8107	phenytoin	<https://orcid.org/0000-0002-7073-9172>|<https://orcid.org/0000-0001-8314-2140>
-OBA:2040131		CHEBI:81278	4-(4-Hydroxyphenyl)-2-butanol	<https://orcid.org/0000-0002-7073-9172>|<https://orcid.org/0000-0001-8314-2140>
-OBA:2040132		CHEBI:82978	paliperidone	<https://orcid.org/0000-0002-7073-9172>|<https://orcid.org/0000-0001-8314-2140>
-OBA:2040133		CHEBI:8378	prednisolone	<https://orcid.org/0000-0002-7073-9172>|<https://orcid.org/0000-0001-8314-2140>
-OBA:2040134		CHEBI:8707	quetiapine	<https://orcid.org/0000-0002-7073-9172>|<https://orcid.org/0000-0001-8314-2140>
-OBA:2040135		CHEBI:8871	risperidone	<https://orcid.org/0000-0002-7073-9172>|<https://orcid.org/0000-0001-8314-2140>
-OBA:2040136		CHEBI:9150	simvastatin	<https://orcid.org/0000-0002-7073-9172>|<https://orcid.org/0000-0001-8314-2140>
-OBA:2040137		CHEBI:9334	sulfasalazine	<https://orcid.org/0000-0002-7073-9172>|<https://orcid.org/0000-0001-8314-2140>
-OBA:2040138		CHEBI:9398	tamsulosin	<https://orcid.org/0000-0002-7073-9172>|<https://orcid.org/0000-0001-8314-2140>
-OBA:2040139		CHEBI:9448	terbinafine	<https://orcid.org/0000-0002-7073-9172>|<https://orcid.org/0000-0001-8314-2140>
-OBA:2040140		CHEBI:9747	Triptolide	<https://orcid.org/0000-0002-7073-9172>|<https://orcid.org/0000-0001-8314-2140>
-OBA:2040141		CHEBI:9943	venlafaxine	<https://orcid.org/0000-0002-7073-9172>|<https://orcid.org/0000-0001-8314-2140>
+defined_class	defined_class_name	chemical	chemical_name	orcid
+OBA:2040000		CHEBI:10033	warfarin	https://orcid.org/0000-0001-8314-2140
```

Comment:
What about the other orcids? Seems like you are now only allowing one?

Reply:
Until the technical issue of adding orcids as data_list_vars is resolved, I can only add one orcid. I will make a ticket and raise this at the tech. team meeting on Monday.

Reply:
Excellent thank you!

Reply:
Contributor ORCIDs are fixed now.